icons/scan-heart.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M17 3h2a2 2 0 0 1 2 2v2" />
+  <path d="M21 17v2a2 2 0 0 1-2 2h-2" />
+  <path d="M3 7V5a2 2 0 0 1 2-2h2" />
+  <path d="M7 21H5a2 2 0 0 1-2-2v-2" />
+  <path d="M8.203 9.27a2.4 2.4 0 0 1 3.441 0l.36.34.34-.34a2.4 2.4 0 0 1 3.449-.01c.95.95 1 2.53-.2 3.74l-3.59 3.55-3.6-3.55c-1.2-1.2-1.14-2.8-.2-3.73" />
```

Comment:
This is off grid, maybe we can adapt the [`heart` from Lucide12](https://lucide-studio.vercel.app/edit?value=%3Csvg+width=%2212%22+height=%2212%22%3E%3Cpath+d=%22M10.324+6.556A2.75+2.75+0+1+0+6+3.168a2.75+2.75+0+1+0-4.324+3.388l3.57+4.1a1+1+0+0+0+1.508.001z%22+/%3E%3C/svg%3E) here?

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M11.246+16.657a1+1+0+0+0+1.508+0l3.57-4.101A2.75+2.75+0+1+0+12+9.168a2.75+2.75+0+1+0-4.324+3.388z%22+/%3E%3Cpath+d=%22M17+3h2a2+2+0+0+1+2+2v2%22+/%3E%3Cpath+d=%22M21+17v2a2+2+0+0+1-2+2h-2%22+/%3E%3Cpath+d=%22M3+7V5a2+2+0+0+1+2-2h2%22+/%3E%3Cpath+d=%22M7+21H5a2+2+0+0+1-2-2v-2%22+/%3E"><img alt="icons" width="200px" src="https://lucide.dev/api/gh-icon/PHN2ZwogIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIKICB3aWR0aD0iMjQiCiAgaGVpZ2h0PSIyNCIKICB2aWV3Qm94PSIwIDAgMjQgMjQiCiAgZmlsbD0ibm9uZSIKICBzdHJva2U9ImN1cnJlbnRDb2xvciIKICBzdHJva2Utd2lkdGg9IjIiCiAgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIgogIHN0cm9rZS1saW5lam9pbj0icm91bmQiCj4KICA8cGF0aCBkPSJNMTEuMjQ2IDE2LjY1N2ExIDEgMCAwIDAgMS41MDggMGwzLjU3LTQuMTAxQTIuNzUgMi43NSAwIDEgMCAxMiA5LjE2OGEyLjc1IDIuNzUgMCAxIDAtNC4zMjQgMy4zODh6IiAvPgogIDxwYXRoIGQ9Ik0xNyAzaDJhMiAyIDAgMCAxIDIgMnYyIiAvPgogIDxwYXRoIGQ9Ik0yMSAxN3YyYTIgMiAwIDAgMS0yIDJoLTIiIC8+CiAgPHBhdGggZD0iTTMgN1Y1YTIgMiAwIDAgMSAyLTJoMiIgLz4KICA8cGF0aCBkPSJNNyAyMUg1YTIgMiAwIDAgMS0yLTJ2LTIiIC8+Cjwvc3ZnPg==.svg"/><br/>Open lucide studio</a>